cranelift/codegen/meta/src/cdsl/instructions.rs

```diff
@@ -171,52 +171,63 @@ impl InstructionBuilder {
         self
     }
 
+    /// Mark this instruction as a block terminator.
     #[allow(clippy::wrong_self_convention)]
-    pub fn is_terminator(mut self, val: bool) -> Self {
-        self.is_terminator = val;
+    pub fn is_terminator(mut self) -> Self {
```

Comment:
If I were to see a function called is_terminator() without any context I would assume it returns a bool. Maybe name is make_terminator(), set_terminator() or terminator() instead?

Reply:
Yeah, I think that's a good idea. We're already disabling the clippy lint for the name, so maybe now's the time to fix this.

Reply:
I think I like `terminator` best of these alternative names, but that's not so good for renaming `is_return`. Other alternatives I'm thinking about are `terminator_inst` or `terminates_block`/`returns`.

Reply:
I picked `return_`, which I agree is not the nicest option. I like `returns`, and will switch to that. What about turning the others into actions as well? (`terminates`, `branches`, `returns`)

Reply:
Yeah, I like `branches`. I'm not certain about `terminates`, so I suggested `terminates_block`. This is more bikeshedding than I intended to get into though and I'm happy enough with any of these options. :smile: